sdk/communication/azure-communication-administration/CHANGELOG.md

```diff
@@ -1,11 +1,9 @@
 # Release History
 
 ## 1.0.0-beta.4 (Unreleased)
-### Added
-- Added support for Azure Active Directory Authentication.
-
 ### Breaking Changes
 - CommunicationIdentityClient and CommunicationIdentityAsyncClient is moved to a new package, `azure-communication-identity`.
+- PhoneNumbersClient and PhoneNumbersAsyncClient is moved to a new package, `azure-communication-phonenumbers`.
```

Comment:
This file had been updated after Feb. 9th release. Please rebase from master and move this line to beta.5 section